error-handling-proto/src/main/proto/bakdata/kafka/proto/v1/deadletter.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package bakdata.kafka.proto.v1;
+
+import "google/protobuf/wrappers.proto";
+
+option java_package = "com.bakdata.kafka.proto.v1";
+option java_multiple_files = true;
+
+message DeadLetter {
```

Comment:
Hm, I'm thinking whether we should use a different name. The package differs from Avro but when we integrate it into the dead-letter-analyzer, we will not be able to import both. What do you think?

Reply:
I'll prefix them, also confuses the IDE less.

Reply:
For better backwards compatibility please leave Avro as it was

Reply:
@mkcode92 I think you missed that